spritz/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        jcenter()
+
```

Comment:
Extra whiteline

Reply:
Done in 2f5e61a472fd70180ccdad9a6c9ea95810b4dd75.